### PR TITLE
Switch to Standard Term Support (STS) releases

### PR DESCRIPTION
**Pull request by richlander**
The "Current" term is switching to "Short Term Support (STS)". It nicely and intuitively contrasts with "Long Term Support (LTS)". Nothing else is changing (beyond the name).

Relates to:

- https://github.com/dotnet/designs/blob/main/accepted/2022/dotnet-release-labels.md
- https://github.com/dotnet/core/pull/7499

### CODE REVIEW

**Comment by richlander** [comment]
I'll merge this in a bit. I'm going to publish a breaking change announcement, first.

**Comment by richlander** [comment]
How about you be the one to merge this PR, @mairaw?

**Comment by richlander** [comment]
I also need to update docs. I'll create a PR on that next week. We can merge that one coordinated with the website.

**Comment by mairaw** [comment]
Sounds good @richlander. I didn't have time last week to create the PR.

**Comment by richlander** [comment]
Not a problem. We're not in a rush. I'll get the docs PR in place first. That should set the stage for getting the website updated. I inserted a mention of STS in the Preview 5 blog post, which you may have seen in draft.

**Comment by leo-costa** [comment]
Good change. Too many folks thinking that LTS = Stable.

**Comment by mairaw** [comment]
what is the plan for dotnet install scripts?
/cc @YuliiaKovalova

**Comment by mairaw** [comment]
We probably need to do similar changes on docs /cc @adegeo

**Comment by richlander** [comment]
@mairaw -- I already added STS on this page: https://docs.microsoft.com/dotnet/core/introduction#support.

Perhaps we can just do this as a rolling thunder type thing and merge this PR now?

File: release-policies.md [hunk 1]
```diff
@@ -1,52 +1,54 @@
...
 ## Release cadence
 
-New major .NET versions are released annually in November. .NET 5.0 was the first version to be released according to this schedule. Minor releases may also be released, with no pre-defined or regular schedule.
+New major .NET versions are released annually in November, at [.NET Conf](https://www.dotnetconf.net/).
 
-## Release types
+Patch updates are released monthly on the second tuesday of each month, also known as Patch Tuesday.
 
-Each .NET release is defined (prior to initial release) as either **Preview**, **Current**, or **Long Term Support (LTS)**. The difference is support time frame, as defined below:
+Minor versions of the .NET SDK are releases approximately quarterly. These are known as [feature bands](https://docs.microsoft.com/en-us/dotnet/core/releases-and-support#feature-bands-sdk-only).
```
Comment (Falco20019) [inline release-policies.md:11]: Suggestion: `are releases approximately quarterly` sounds strange to me. Maybe `release approximately quartlery` would be better.

Comment (gewarren) [reply release-policies.md:11]:
```suggestion
Minor versions of the .NET SDK are released approximately quarterly. These are known as [feature bands](https://docs.microsoft.com/dotnet/core/releases-and-support#feature-bands-sdk-only).
```

File: release-policies.md [hunk 2]
```diff
@@ -1,52 +1,54 @@
...
-The .NET team uses the following policies for [.NET releases](releases.md). They define what you can expect when you use a given .NET release.
+The .NET team uses the following policies for [.NET releases](releases.md).
 
 ## Release cadence
 
-New major .NET versions are released annually in November. .NET 5.0 was the first version to be released according to this schedule. Minor releases may also be released, with no pre-defined or regular schedule.
+New major .NET versions are released annually in November, at [.NET Conf](https://www.dotnetconf.net/).
 
-## Release types
+Patch updates are released monthly on the second tuesday of each month, also known as Patch Tuesday.
```
[the remark this answers is not in the document]
Comment (mairaw) [reply release-policies.md:9]:
```suggestion
Patch updates are released monthly on the second Tuesday of each month, also known as Patch Tuesday.
```

File: release-notes/7.0/README.md [hunk 3]
```diff
@@ -1,7 +1,6 @@
 # .NET 7
 
-[.NET 7](https://devblogs.microsoft.com/dotnet/announcing-dotnet-7-preview-2/) will be a [current release](../../release-policies.md) and will be supported for 18 months, from November 2022 to May 14, 2024. It is [supported by Microsoft](../../microsoft-support.md) on [multiple operating systems](supported-os.md).
-
+[.NET 7](https://devblogs.microsoft.com/dotnet/announcing-dotnet-7-preview-4/) is an [STS release](../../release-policies.md) and will be supported for 18 months, from November 8th, 2022 to May 14th, 2024. It is [supported by Microsoft](../../microsoft-support.md) on [multiple operating systems](supported-os.md).
```
Comment (gewarren) [inline release-notes/7.0/README.md:3]:
```suggestion
[.NET 7](https://devblogs.microsoft.com/dotnet/announcing-dotnet-7-preview-4/) is a [standard-support release](../../release-policies.md) and will be supported for 18 months, from November 8th, 2022 to May 14th, 2024. It is [supported by Microsoft](../../microsoft-support.md) on [multiple operating systems](supported-os.md).

```

File: release-notes/7.0/supported-os.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 # .NET 7 - Supported OS versions
 
-[.NET 7](README.md) [is supported](https://github.com/dotnet/core/blob/main/microsoft-support.md) on multiple operating systems per their [lifecycle policy](../../os-lifecycle-policy.md).
+[.NET 7](README.md) is an [STS release](../../release-policies.md) and [is supported](../../microsoft-support.md) on multiple operating systems per their lifecycle policy.
```
Comment (gewarren) [inline release-notes/7.0/supported-os.md:3]:
```suggestion
[.NET 7](README.md) is a [standard-support release](../../release-policies.md) and [is supported](../../microsoft-support.md) on multiple operating systems per their lifecycle policy.
```

File: release-policies.md [hunk 5]
```diff
@@ -1,52 +1,54 @@
...
-* **Current** releases are supported for (typically) eighteen months. They are intended for users that want to take advantage of the newest features and improvements and to stay on the leading edge of .NET innovation. Current release users need to upgrade to later .NET releases more often to stay in support.
-* **LTS** releases are supported for three years. They are intended for users that want the stability and lower cost of maintaining an application on a single (major.minor) .NET version for an extended period.
+## Release types
 
-LTS and Current releases have many similarities. The .NET team follows the same software engineering and release processes for both release types, including for security, compatibility, and reliability. Both releases may contain major new features and breaking changes. The .NET team aspires to enable straightforward migration from one release to another (LTS or Current, in either direction), and has processes in place to achieve that intention.
+Each .NET release is defined as either  **Standard Support** or **Long Term Support (LTS)**, at the beginning of the release.
```
Comment (gewarren) [inline release-policies.md:15]:
```suggestion
Each .NET release is defined as either  **standard support** or **long term support (LTS)**, at the beginning of the release.
```

File: release-policies.md [hunk 6]
```diff
@@ -1,52 +1,54 @@
...
 
-### Full support
+## Support phases
 
-During the full support period, .NET releases are updated to improve functional capabilities and mitigate security vulnerabilities.
+.NET releases go through multiple support phases, with varying support levels.
 
-Functional improvements are typically very targeted, and may address the following:
+* **Preview** releases are not supported but are offered for public testing and for the opportunity to give feedback.
+* **Go-live** support enables users to deploy a pre-release build in production, and are supported by Microsoft. These are typically Release Candidate (RC) releases.
```
Comment (gewarren) [inline release-policies.md:31]:
```suggestion
* **Go-live** support enables users to deploy a pre-release build in production, and are supported by Microsoft. These are typically release candidate (RC) releases.
```

File: release-policies.md [hunk 7]
```diff
@@ -1,52 +1,54 @@
...
+* **Active** support is provided for the majority of the period after a release is Generally Available (GA). Functional and security improvements will be provided, including support for new operating system versions.
```
Comment (gewarren) [inline release-policies.md:32]:
```suggestion
* **Active** support is provided for the majority of the period after a release is generally available (GA). Functional and security improvements will be provided, including support for new operating system versions.
```

File: releases.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 # .NET Releases
 
-The .NET team releases new major versions of .NET annually, each November. Releases are either Long Term Support (LTS) or Current, the two .NET release types, and transition from full support, to maintenance and finally to end-of-life on a defined schedule, per [.NET release policies][policies]. .NET releases are [supported by Microsoft](microsoft-support.md) on [multiple operating systems](os-lifecycle-policy.md) and hardware architectures.
+The .NET team releases new major versions of .NET annually, each November. Releases are either Long Term Support (LTS) or Standard Support, the two .NET release types, and transition from full support, to maintenance and finally to end-of-life on a defined schedule, per [.NET release policies][policies]. .NET releases are [supported by Microsoft](microsoft-support.md) on [multiple operating systems](os-lifecycle-policy.md) and hardware architectures.
```
Comment (gewarren) [inline releases.md:3]:
```suggestion
The .NET team releases new major versions of .NET annually, each November. Releases are either long term support (LTS) or standard support, the two .NET release types, and transition from full support, to maintenance and finally to end-of-life on a defined schedule, per [.NET release policies][policies]. .NET releases are [supported by Microsoft](microsoft-support.md) on [multiple operating systems](os-lifecycle-policy.md) and hardware architectures.
```

**Review by gewarren — APPROVED** [review]
I uncapitalized the support terms since they aren't product names or anything like that. From https://styleguides.azurewebsites.net/StyleGuide/Read?id=2696&topicid=25357:

"If it’s not a brand, product, service, or app it’s lowercase."

**Comment by mairaw** [comment]
I think standardizing on title case or all lowercase would be a question for @jamshedd and @richlander. Let me know the preference and I'll adjust accordingly on all PRs. It needs to be consistent.

**Comment by jamshedd** [comment]
> I think standardizing on title case or all lowercase would be a question for @jamshedd and @richlander. Let me know the preference and I'll adjust accordingly on all PRs. It needs to be consistent.

I think we need to keep this Long Term Support (LTS) and Standard Support with that capitalization in our content/website/etc. the one exception to this is the releases.json where the enums already use all lower case, so we'd keep that to be consistent with the rest of the file.

**Comment by gewarren** [comment]
> I think we need to keep this Long Term Support (LTS) and Standard Support with that capitalization in our content/website/etc. the one exception to this is the releases.json where the enums already use all lower case, so we'd keep that to be consistent with the rest of the file.

I'll go with that, but it does go against the [Cloud & AI style guide](https://styleguides.azurewebsites.net/StyleGuide/Read?id=2696&topicid=25357). Also if we lower-case "standard support" it becomes less of a confusion with ".NET Standard", which is always capitalized.

**Comment by mairaw** [comment]
Thanks for sharing that guidance @gewarren. We can standardize on lowercase across all materials. We just need to do a pass to ensure is all consistent. But I'll give the final word to @jamshedd.

**Comment by jamshedd** [comment]
> > I think we need to keep this Long Term Support (LTS) and Standard Support with that capitalization in our content/website/etc. the one exception to this is the releases.json where the enums already use all lower case, so we'd keep that to be consistent with the rest of the file.
> 
> I'll go with that, but it does go against the [Cloud & AI style guide](https://styleguides.azurewebsites.net/StyleGuide/Read?id=2696&topicid=25357). Also if we lower-case "standard support" it becomes less of a confusion with ".NET Standard", which is always capitalized.

Thanks for the pointer Genevieve. I don't think there's a real risk of anyone confusion .NET Standard (the target) and Standard Support (the support policy). I would stick with the capitalization as it exists.

File: release-policies.md [hunk 9]
```diff
@@ -1,52 +1,54 @@
...
 
-During the full support period, .NET releases are updated to improve functional capabilities and mitigate security vulnerabilities.
+.NET releases go through multiple support phases, with varying support levels.
 
-Functional improvements are typically very targeted, and may address the following:
+* **Preview** releases are not supported but are offered for public testing and for the opportunity to give feedback.
+* **Release Candidate** releases have a go-live support that enables users to deploy a pre-release build in production and are supported by Microsoft.
+* **Active** support is provided for the majority of the period after a release is generally available (GA). Functional and security improvements will be provided, including support for new operating system versions.
+* **Maintenance** support is provided for the last six months of support. Improvements are limited to security fixes. Support for new operating system versions will be provided on a best-effort basis.
+* **End of support** or end-of-life (EOL) marks the point where a release is no longer supported.
```
Comment (jamshedd) [inline release-policies.md:34]:
```suggestion
* **End of support** or end-of-life (EOL) marks the point where a release no longer receives servicing updates and is no longer supported.
```

File: release-policies.md [hunk 10]
```diff
@@ -1,52 +1,54 @@
...
+## Servicing
 
-### Maintenance support
+Improvements are released as full re-releases of the product, called "patch releases". Patch releases are cumulative. Patches are released on the Microsoft "Patch Tuesday" (second Tuesday of each month), however there is no guarantee that there will be a .NET release on any given Patch Tuesday.
 
-During the maintenance support period, .NET releases are updated to mitigate security vulnerabilities, only.
+Patches are announced in [release notes](release-notes/README.md), on the [.NET blog](https://devblogs.microsoft.com/dotnet/category/maintenance-and-updates/), and [dotnet/announcements](https://github.com/dotnet/announcements/labels/Monthly-Update).
 
-The maintenance support period is the final 6 months of support for any release (Current or LTS). After the maintenance period ends, the release is out of support.
+Breaking changes are not accepted during servicing, except (in the very rare case) to resolve critical issues, such as a security vulnerability.
```
Comment (jamshedd) [inline release-policies.md:44]:
```suggestion
Breaking changes are not accepted during servicing, except (in the very rare case) to resolve critical issues, such as a security vulnerability. Breaking changes are documented.
```

**Review by jamshedd — CHANGES_REQUESTED** [review]

File: release-policies.md [hunk 11]
```diff
@@ -1,52 +1,54 @@
...
 
-### End of support
+## End of support
 
-"End of support", "out of support", or "end of life" refers to the date after which fixes, updates, or technical assistance are no longer provided. As the end of support nears for a given .NET version, we recommend that you move to a newer .NET version, and reduce/remove your use of the given .NET version. After support ends, we recommend that you uninstall a given .NET version if you are no longer using it, or install the latest patch, and accelerate your plans to remove your use of that .NET version.
+As the end of support nears for a given .NET version, we recommend that you move to a newer .NET version, and reduce/remove your use of the given .NET version. After support ends, we recommend that you uninstall a given .NET version if you are no longer using it, or install the latest patch, and accelerate your plans to remove your use of that .NET version.
```
[the remark this answers is not in the document]
Comment (richlander) [reply release-policies.md:48]: What wording would you suggest instead?

File: release-policies.md [hunk 12]
```diff
@@ -1,52 +1,54 @@
...
 
-### Full support
+## Support phases
 
-During the full support period, .NET releases are updated to improve functional capabilities and mitigate security vulnerabilities.
+.NET releases go through multiple support phases, with varying support levels.
 
-Functional improvements are typically very targeted, and may address the following:
+* **Preview** releases are not supported but are offered for public testing and for the opportunity to give feedback.
+* **Release Candidate** releases have a go-live support that enables users to deploy a pre-release build in production and are supported by Microsoft.
```
Comment (leecow) [inline release-policies.md:31]: "Release Candidate" should be changed to "Go-Live"

File: release-policies.md (hunk 9 above)
Comment (leecow) [inline release-policies.md:34]: "End of support" should be "End of life"

File: release-policies.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (leecow) [reply release-policies.md:48]: Term should remain "end of life"

**Review by leecow — CHANGES_REQUESTED** [review]

File: release-policies.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (jamshedd) [reply release-policies.md:48]: This isn't about EOL v/s EOS.

[the remark this answers is not in the document]
Comment (jamshedd) [reply release-policies.md:48]:
```suggestion
As the end of support nears for a given .NET version, we strongly recommend you move to a newer, supported version of .NET. .NET releases that have reached end of support do not get security or other patches, so continuing to use such a version may expose you to unpatched security vulnerabilities.
```

File: release-notes/6.0/supported-os.md [hunk 13]
```diff
@@ -1,8 +1,8 @@
 # .NET 6 - Supported OS versions
 
-[.NET 6](README.md) [is supported](https://github.com/dotnet/core/blob/main/microsoft-support.md) on multiple operating systems per their [lifecycle policy](../../os-lifecycle-policy.md).
+[.NET 6](README.md) is an [LTS release](../../release-policies.md) and [is supported](../../microsoft-support.md) on multiple operating systems per their lifecycle policy.
```
[the remark this answers is not in the document]
Comment (richlander) [reply release-notes/6.0/supported-os.md:3]: I don't see it that way. The wording seems accurate and intuitive.

> implies we follow the OS lifecycle (like Framework) which we don't

Almost no reader will come to this conclusion.

File: release-policies.md (hunk 12 above)
Comment (richlander) [inline release-policies.md:31]:
```suggestion
* **Go-Live** releases are supported by Microsoft in production. These are typically our release candidate builds just before the generally available release.
```

File: release-policies.md (hunk 9 above)
Comment (richlander) [reply release-policies.md:34]: I don't see that this addition is necessary. It is fully implied and called out in full detail in a later section.

File: release-notes/7.0/README.md [hunk 14]
```diff
@@ -1,7 +1,6 @@
 # .NET 7
 
-[.NET 7](https://devblogs.microsoft.com/dotnet/announcing-dotnet-7-preview-2/) will be a [current release](../../release-policies.md) and will be supported for 18 months, from November 2022 to May 14, 2024. It is [supported by Microsoft](../../microsoft-support.md) on [multiple operating systems](supported-os.md).
-
+[.NET 7](https://devblogs.microsoft.com/dotnet/announcing-dotnet-7-preview-4/) is a [Standard Support release](../../release-policies.md) and will be supported for 18 months, from November 8th, 2022 to May 14th, 2024. It is [supported by Microsoft](../../microsoft-support.md) on [multiple operating systems](supported-os.md).
```
Comment (richlander) [inline release-notes/7.0/README.md:3]:
```suggestion
[.NET 7](https://devblogs.microsoft.com/dotnet/announcing-dotnet-7-preview-4/) is a [Standard Term Support release](../../release-policies.md) and will be supported for 18 months, from November 8th, 2022 to May 14th, 2024. It is [supported by Microsoft](../../microsoft-support.md) on [multiple operating systems](supported-os.md).

```

File: release-notes/7.0/supported-os.md [hunk 15]
```diff
@@ -1,8 +1,8 @@
 # .NET 7 - Supported OS versions
 
-[.NET 7](README.md) [is supported](https://github.com/dotnet/core/blob/main/microsoft-support.md) on multiple operating systems per their [lifecycle policy](../../os-lifecycle-policy.md).
+[.NET 7](README.md) is a [Standard Support release](../../release-policies.md) and [is supported](../../microsoft-support.md) on multiple operating systems per their lifecycle policy.
```
Comment (richlander) [inline release-notes/7.0/supported-os.md:3]:
```suggestion
[.NET 7](README.md) is a [Standard Term Support release](../../release-policies.md) and [is supported](../../microsoft-support.md) on multiple operating systems per their lifecycle policy.
```

File: release-policies.md [hunk 16]
```diff
@@ -1,52 +1,54 @@
...
-* **Current** releases are supported for (typically) eighteen months. They are intended for users that want to take advantage of the newest features and improvements and to stay on the leading edge of .NET innovation. Current release users need to upgrade to later .NET releases more often to stay in support.
-* **LTS** releases are supported for three years. They are intended for users that want the stability and lower cost of maintaining an application on a single (major.minor) .NET version for an extended period.
+## Release types
 
-LTS and Current releases have many similarities. The .NET team follows the same software engineering and release processes for both release types, including for security, compatibility, and reliability. Both releases may contain major new features and breaking changes. The .NET team aspires to enable straightforward migration from one release to another (LTS or Current, in either direction), and has processes in place to achieve that intention.
+Each .NET release is defined as either **Standard Support** or **Long Term Support (LTS)**, at the beginning of the release.
```
Comment (richlander) [inline release-policies.md:15]:
```suggestion
Each .NET release is defined as either **Standard Term Support (STS)** or **Long Term Support (LTS)**, at the beginning of the release.
```

File: release-policies.md [hunk 17]
```diff
@@ -1,52 +1,54 @@
...
 
-## Servicing
+The release types:
 
-.NET releases are supported -- during the servicing period -- according to the following policies. Servicing policies are the same for LTS and Current releases.
+* **Standard Support** releases are supported for eighteen months, released in even-numbered years. They are intended for users that want to take advantage of the newest features and improvements and to stay on the leading edge of .NET innovation.
+* **LTS** releases are supported for three years, released in odd-numbered years. They are intended for users that want the stability and lower cost of maintaining an application for an extended period, only needing to upgrade their .NET version for security patches.
```
Comment (richlander) [inline release-policies.md:20]:
```suggestion
* **STS** releases are supported for eighteen months, released in even-numbered years. They are intended for users that want to take advantage of the newest features and improvements and to stay on the leading edge of .NET innovation.
* **LTS** releases are supported for three years, released in odd-numbered years. They are intended for users that want the stability and lower cost of maintaining an application for an extended period, only needing to upgrade their .NET version for security patches.
```

File: release-policies.md [hunk 18]
```diff
@@ -1,52 +1,54 @@
...
 
-A single "bug bar" is used to decide if a change is warranted and safe for servicing updates. A given fix is often applied to multiple [servicing branches](https://github.com/dotnet/core/blob/main/daily-builds.md#servicing-releases), independent of release type. Breaking changes are not accepted during servicing, except (in the very rare case) to resolve a security vulnerability.
+Note: **Standard** releases were previously called **Current**.
```
Comment (richlander) [inline release-policies.md:22]:
```suggestion
Note: **Standard Term Support** releases were previously called **Current**.
```

File: release-policies.md [hunk 19]
```diff
@@ -1,52 +1,54 @@
...
 
-Improvements are released as "patches". Patch releases are cumulative. Patches are released on the Microsoft "Patch Tuesday" (second Tuesday of each month), however there is no guarantee that there will be a .NET release on any given Patch Tuesday. Patches are announced on the [.NET blog](https://devblogs.microsoft.com/dotnet/). A digest of monthly releases is published to [dotnet/announcements](https://github.com/dotnet/announcements/labels/Monthly-Update).
+LTS and Standard Support releases differ only by support duration. The .NET team follows the same software engineering and release processes for both release types, including for security, compatibility, and reliability. Both releases may contain major new features and breaking changes. The .NET team aspires to enable straightforward migration from one release to another, independent of release type.
```
Comment (richlander) [inline release-policies.md:24]:
```suggestion
LTS and STS releases differ only by support duration. The .NET team follows the same software engineering and release processes for both release types, including for security, compatibility, and reliability. Both releases may contain major new features and breaking changes. The .NET team aspires to enable straightforward migration from one release to another, independent of release type.
```

File: releases.md (hunk 8 above)
Comment (richlander) [inline releases.md:3]:
```suggestion
The .NET team releases new major versions of .NET annually, each November. Releases are either Long Term Support (LTS) or Standard Term Support (STS), the two .NET release types, and transition from full support, to maintenance and finally to end-of-life on a defined schedule, per [.NET release policies][policies]. .NET releases are [supported by Microsoft](microsoft-support.md) on [multiple operating systems](os-lifecycle-policy.md) and hardware architectures.
```

File: release-notes/7.0/README.md (hunk 3 above)
Comment (richlander) [reply release-notes/7.0/README.md]: Note that Windows also does caps for LTSC -> https://learn.microsoft.com/en-us/windows/whats-new/ltsc/

Comment (gewarren) [reply release-notes/7.0/README.md]: They are wrong too 😂

**Pull request merged**